src/manifolds/GeneralUnitaryMatrices.jl

```diff
@@ -342,17 +342,22 @@ function get_coordinates_orthogonal(M::GeneralUnitaryMatrices{n,ℝ}, p, X, N) w
     return get_coordinates_orthogonal!(M, Y, p, X, N)
 end
 
+function get_coordinates_orthogonal!(::GeneralUnitaryMatrices{1,ℝ}, Xⁱ, p, X, ::RealNumbers)
```

Comment:
`Xⁱ` that is passed can only be an empty vector, and the correct returned value is also an empty vector. There should be no 0 because there are no coordinates.

Reply:
ah right, now I confused this with the other place where I said we should return `[]`.